azure-pipelines.yml

```diff
@@ -12,13 +12,29 @@ jobs:
       organization: 'sonarsource-cfamily-examples'
       scannerMode: 'CLI'
       configMode: 'file'
+      extraProperties: |
+        sonar.cfamily.vscoveragexml.reportsPath=$(System.DefaultWorkingDirectory)/CoverageResults/*/*.xml
   - pwsh: |
       curl --create-dirs -sSLo $HOME\.sonar\build-wrapper-win-x86.zip ${{variables.BUILD_WRAPPER_DOWNLOAD_URL}}
       unzip -o $HOME\.sonar\build-wrapper-win-x86.zip -d $HOME\.sonar\
     displayName: Download and install build wrapper
   - pwsh: |
       $path = vswhere -latest -requires Microsoft.Component.MSBuild -find MSBuild\**\Bin\MSBuild.exe | select-object -first 1
       & "$HOME\.sonar\build-wrapper-win-x86\build-wrapper-win-x86-64.exe" --out-dir build_wrapper_output_directory $path sonar_scanner_example.vcxproj /t:rebuild /nodeReuse:false
+      # Execute the tests to generate the coverage statistics
+      & $path test1\test1.vcxproj
+      & $path test2\test2.vcxproj
     workingDirectory: .
-    displayName: Build in build-wrapper
+    displayName: Build in build-wrapper and tests
+  - task: VSTest@2
+    inputs:
+      testSelector: 'testAssemblies'
+      testAssemblyVer2: |
+        test1\*\test1.dll
+        test2\*\test2.dll
+      searchFolder: '$(System.DefaultWorkingDirectory)'
+      resultsFolder: '$(System.DefaultWorkingDirectory)/CoverageResults'
+      otherConsoleOptions: '/collect:"Code Coverage;Format=Xml"' # Necessary to have an XML coverage report
+      codeCoverageEnabled: true
+      testRunTitle: 'Run tests'
```

Comment:
Does this run the tests? What do lines 25 and 26 do then?

Reply:
Yes, this runs the tests
l.25 and l.26 build the two test projects